python/pyspark/sql/functions.py

```diff
@@ -1414,7 +1414,7 @@ def __init__(self, func, returnType, name=None):
     def _create_judf(self, name):
         f, returnType = self.func, self.returnType  # put them in closure `func`
         func = lambda _, it: map(lambda x: returnType.toInternal(f(*x)), it)
-        ser = AutoBatchedSerializer(PickleSerializer())
+        ser = BatchedSerializer(PickleSerializer(), 100)
```

Comment:
Can we pull this out in a constant? And also the same value in the Python, and put a comment on each saying that they have to equal? It's very dangerous if this value goes out of sync.